utils/evaluate_utils.py

```diff
@@ -274,22 +313,39 @@ def model_confusion_matrix(
         conf_mat, columns=log_reg_model.classes_, index=log_reg_model.classes_
     )
 
-    # display confusion matrix
-    plt.figure(figsize=(10, 10))
-    ax = sns.heatmap(data=conf_mat, annot=True, fmt=".0f", cmap="viridis", square=True)
-    ax = plt.xlabel("Predicted Label")
-    ax = plt.ylabel("True Label")
-    ax = plt.title("Phenotypic Class Predicitions")
-    plt.show()
+    # create confusion matrix figure on ax that is given or make new ax
```

Comment:
Could `ax` be initialized if not already provided, possibly eliminating the need for some of the duplicated code below? The same may apply around line 388. I recognize this may not work for the purposes you have in mind, so feel free to ignore this comment if apt.

Reply:
I think this would make more sense as it follows the `sns` methodology of creating a new `ax` if it doesn't already exist.